opm/parser/eclipse/Deck/UDAValue.hpp

```diff
@@ -64,6 +64,11 @@ class UDAValue {
 
     bool operator==(const UDAValue& other) const;
     bool operator!=(const UDAValue& other) const;
+
+    bool is_numeric() { return numeric_value; }
+    double get_double() { return double_value; }
```

Comment:
The `get_double()` and `get_str()` should not be necessary - you should be able to use the templated `get<double>()` and `get<std::string>()`?

Reply:
Yes thanks,  I did not notice that a get function already was in place